#goal definition
int32 motion_cmd ##home_pose, search_pose, grasp_object, put_object, pull_shelf, push_shelf, test_marker
int32 object_id  #0:kimbap1 1:kimbap2 2:sanwich 3:hambug_thin 4:hambug_fat 5:coffee 6:lunchbox
int32 tool_id    #0:gripper 1:suction
int32 shelf_id   #1:bottom,2:middle,3:top floor
int32 sub_cmd    #
float64 pose_x
float64 pose_y
float64 pose_z
float64 ori_r #roll
float64 ori_p #pitch
float64 ori_y #yaw
---
#result definition
int32 result_flag #0:fail 1:success
float64 pose_x
float64 pose_y
float64 pose_z
float64 ori_r #roll
float64 ori_p #pitch
float64 ori_y #yaw
---
#feedback
int32 dummy
